README.md

```
@@ -55,7 +55,30 @@ No providers.
+make test-unit
+
+# Automation-based integration tests
+make test-integration
+
+# Mutation analysis (time-consuming)
+make test-mutation
+```
+
+You can also execute `make test` to run the structural, unit, and integration checks as a batch.
+
+## Security
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)